BlotoutAnalytics/Utilities/BOServerDataConverter.m

```diff
@@ -58,10 +58,13 @@ +(NSDictionary*)recordAppInformation {
       [appInfo setObject:[NSNumber numberWithInt:timeoOffset] forKey:@"timeZoneOffset"];
       
       //UIWebView Depriciated , need to handle it with WKWebView
+      dispatch_async(dispatch_get_main_queue(), ^{
       UIWebView* webView = [[UIWebView alloc] initWithFrame:CGRectZero];
```

Comment:
We can have another ticket to get eliminate this UIWebView.

Reply:
ok, sure